### PR TITLE
getResources --> getAllResources and getActiveResources queries

### PR DESCRIPTION
**Pull request by mk4111**
See https://github.com/emfoundation/ce100-app/issues/645 and #689

We want to be able to display active resources to primary users, but both active+inactive resources to admins.

This PR updates our queries and tests to accommodate for this.

### CODE REVIEW

**Comment by SimonLab** [comment]
@mk4111 don't forget to update the version to be able to publish the package on npm :+1:

File: lib/index.js [hunk 1]
```diff
@@ -68,14 +68,27 @@ function register (server, options, next) {
     });
   };
 
-  var getResources = function(cb) {
-    query(queries.getResources(), pool, function(err, res) { //eslint-disable-line
+  var getResources = function(active, cb) {
+    if (active) {
+      console.log('active only returned');
```
Comment (SimonLab) [inline lib/index.js]: console.log to be deleted

**Comment by mk4111** [comment]
@SimonLab - have removed loggy log :evergreen_tree: !

**Review by SimonLab — APPROVED** [review]
Looks good @mk4111 thanks :+1:

**Pull request merged**